### ISSUE TITLE
Why initiate ROIHead's classification layer stddev to 0.001?

### ISSUE THREAD

**Issue by ZHANGHeng19931123** [chenyuntc/simple-faster-rcnn-pytorch#105]
**File:** model/faster_rcnn_vgg16.py,
**Line 109:** normal_init(self.cls_loc, 0, 0.001)

Why initiate classification layer stddev to 0.001 while in other places (**file** faster_rcnn_vgg16.py **line** 110 and **file** region_proposal_network.py **line** 58, 59, 60) they are set to 0.01?

**Comment by yiran7324** [comment]
It is a part of [Fast RCNN](https://arxiv.org/abs/1504.08083).

In **SGD hyper-parameters**:

> The fully connected layers used for softmax classification and bounding-box regression are initialized from zero-mean Gaussian distributions with stan- dard deviations **0.01** and **0.001**, respectively. Biases are ini- tialized to 0. All layers use a per-layer learning rate of 1 for weights and 2 for biases and a global learning rate of 0.001. When training on VOC07 or VOC12 trainval we run SGD for 30k mini-batch iterations, and then lower the learning rate to 0.0001 and train for another 10k iterations. When we train on larger datasets, we run SGD for more iterations, as described later. A momentum of 0.9 and parameter decay of 0.0005 (on weights and biases) are used.
